src/lib/apollo/initApolloServer.js

```diff
@@ -11,17 +12,34 @@ if (!process.browser) {
   global.fetch = fetch;
 }
 
+// error logging for Apollo Link
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors) {
+    graphQLErrors.forEach(({ message, locations, path }) => {
+      // eslint-disable-next-line no-console
+      console.log(`[GraphQL error]: Message: ${message}, Location: ${JSON.stringify(locations)}, Path: ${JSON.stringify(path)}`);
```

Comment:
could we use the logger util for something like this?
https://github.com/reactioncommerce/reaction-next-starterkit/blob/master/src/lib/logger/index.js

Reply:
Well, maybe... but that logger is using `chalk` (for terminal colors) and these particular errors can happen on the server _and_ the client.  I think we probably need a more isomorphic solution for this use case.  We could certainly use [Bunyan](https://github.com/trentm/node-bunyan) for that.  Then we'd have all of the same API's we're used to in Reaction (for piping logs to other places, etc).

Reply:
yeah, I wasn't sure if our `lib/logger` would work in these cases. I'm good with just doing the `console.error` here for now and maybe we revisit our logger soon in another PR